docs/user/metrics/timespan.md

```diff
@@ -67,6 +67,21 @@ assertTrue(Auth.loginTime.testHasValue())
 assertTrue(Auth.loginTime.testGetValue() > 0)
 ```
 
+### Raw API
+
+> **Note**: It's recommended to use the timer API listed above.
```

Comment:
nit: maybe we should slightly rephrase this to something like "The raw API was designed to support a specific set of use-cases. Please consider using the higher level APIs listed above."